lib/new_embed.dart

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2019, the Dart project authors.  Please see the AUTHORS file
```

Comment:
I'd consider moving this into it's own directory inside of `lib/`; that'll help keep the experimental code separate.

Reply:
Done. I've created an `experimental` directory, and moved it in there.